src/Cards/ProductCard.js

```diff
@@ -0,0 +1,95 @@
+import "./ProductCard.css";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import { useMyReducer } from "../stateContext";
+import { useLangContext } from "../AllContext/languageContext";
+
+import { AiOutlineHeart } from "react-icons/ai";
+import { AiFillHeart } from "react-icons/ai";
+
+export const ProductCard = (item) => {
+  const {
+    id,
+    imageSrc,
+    productName,
+    description,
+    price,
+    originalPrice,
+    newFlag
+  } = item;
+  const [wished, setWished] = useState(false);
+  const { dispatch } = useMyReducer();
+  const { language } = useLangContext();
+  return (
+    <div className="card">
+      <div className="product-details">
+        <Link to={`/products/${id}`}>
+          <img className="product-image" alt="product" src={imageSrc} />
+          {newFlag && <div className="float-new">{language.new}</div>}
+        </Link>
+
+        <div className="product-name-and-desc">
+          <Link to={`/products/${id}`}>
+            <h3 className="product-name-style">{productName}</h3>
+            <br />
+            <div className="price-details">
+              <strong>Rs. {price}</strong>
+              <strike className="price-strike-style">
+                Rs. {originalPrice}
+              </strike>
+              <br />
+              <span className="red-text">
+                (
+                {`${Math.floor((originalPrice - price) / 100)}% ${
```

Comment:
You can create a function to get the price, to make it more readable

Reply:
Yeh can do that! Done